pnpm/src/pnpm.ts

```diff
@@ -15,8 +15,13 @@ const argv = process.argv.slice(2)
     break
   }
   case 'install-completion': {
```

Comment:
you said you'll remove install-completion and `uninstall-completion` from the commands.

Reply:
I converted them to plugins then forgot to remove them here. When I remove them, I completely forgot to remove the switch block.